nestkernel/event_delivery_manager.cpp

```diff
@@ -617,13 +617,7 @@ EventDeliveryManager::deliver_events_( const size_t tid, const std::vector< Spik
       {
         if ( spike_data.get_tid() == tid )
         {
-          const size_t syn_id = spike_data.get_syn_id();
-          const size_t lcid = spike_data.get_lcid();
-
-          // non-local sender -> receiver retrieves ID of sender Node from SourceTable based on tid, syn_id, lcid
-          // only if needed, as this is computationally costly
-          se.set_sender_node_id_info( tid, syn_id, lcid );
```

Comment:
I am afraid I don't understand why the event does not have to be properly parametrized anymore. Can you please comment on this?

Reply:
Well, this is the point of this PR, _before_ it wasn't parametrized correctly, because the lcid was actually not correct (it always pointed to the very first connection, instead of the n-th one after the first). This line just got moved from the event delivery manager to the connector.

Reply:
Ah! Thanks! I failed to grasp that the line was *moved*. Thanks.

Reply:
I stumbled across this too. Which piece of code is now equivalent to parameterizing the event with the correct lcid? The changes in the connector seem to only regard the variable names, don't they?

Reply:
Yes, I see now. Thanks.